src/nomadic/realtime/commands.py

```diff
@@ -233,15 +233,37 @@ def get_region_path(region_bed, workspace):
     return region_bed
 
 
-def get_metadata_path(experiment_name, metadata_csv, workspace):
-    if not metadata_csv:
-        metadata_csv = workspace.get_metadata_csv(experiment_name)
-        if not os.path.isfile(metadata_csv):
-            raise click.BadParameter(
-                message=f"Metadata CSV file not found at {metadata_csv}. Did you create your metadata file in `{workspace.get_metadata_dir()}` and does the name match `{experiment_name}`?",
+def get_metadata_path(experiment_name, metadata_path, workspace):
+    if metadata_path is None:
```

Comment:
If the metadata_path is not None, then does it make sense passing through this function? I would suggest we rather had:

```
if metadata_path is None:
  metadata_path = get_metadata_path(experiment_name, workspace)
```

At present, it is confusing to be passing a 'metadata_path' into a function called 'get_metadata_path()'